content/en/docs/setup/_index.md

```diff
@@ -12,84 +12,101 @@ content_template: templates/concept
+| [Nutanix](https://www.nutanix.com/en)         | [Nutanix Karbon](https://www.nutanix.com/products/karbon)             | [Nutanix Karbon](https://www.nutanix.com/products/karbon)       |             |      | [Nutanix AHV](https://www.nutanix.com/products/acropolis/virtualization)         |
+| [OpenShift](https://www.openshift.com)          |[OpenShift Dedicated](https://www.openshift.com/products/dedicated/) and [OpenShift Online](https://www.openshift.com/products/online/)              |        | [OpenShift Container Platform](https://www.openshift.com/products/container-platform/)            |      |  [OpenShift Container Platform](https://www.openshift.com/products/container-platform/)        |[OpenShift Container Platform](https://www.openshift.com/products/container-platform/)
+| [Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE)](https://docs.cloud.oracle.com/iaas/Content/ContEng/Concepts/contengoverview.htm)          | &#x2714;             |   &#x2714;     |             |      |          |
+| [oVirt](https://www.ovirt.org/)         |              |        |             |     | &#x2714;          |
+| [Pivotal Container Service](https://pivotal.io/platform/pivotal-container-service)         |              | &#x2714;       | &#x2714;           |      |          |
+| [Platform9](https://platform9.com/)         | &#x2714;             | &#x2714;      | &#x2714;            |      |   &#x2714;       |&#x2714;
+| [Rancher](https://rancher.com/)         |              |   [Rancher 2.x](https://rancher.com/docs/rancher/v2.x/en/)     |             |  [Rancher Kubernetes Engine (RKE)](https://rancher.com/docs/rke/latest/en/)    |          | [k3s](https://k3s.io/)
+| [StackPoint](https://stackpoint.io/)          | &#x2714;            |  &#x2714;      |             |      |          |
+| [Supergiant](https://supergiant.io/)         |              |&#x2714;        |             |      |          |
+| [SUSE](https://www.suse.com/)        |              | &#x2714;       |             |      |          |
+| [SysEleven](https://www.syseleven.io/)         | &#x2714;             |        |             |      |          |
+| [VEXXHOST](https://vexxhost.com/)         | &#x2714;             | &#x2714;      |             |      |          |
+| [VMware](https://cloud.vmware.com/) | [VMware Cloud PKS](https://cloud.vmware.com/vmware-cloud-pks)              |[VMware Enterprise PKS](https://cloud.vmware.com/vmware-enterprise-pks)        |   [VMware Enterprise PKS](https://cloud.vmware.com/vmware-enterprise-pks)          | [VMware Essential PKS](https://cloud.vmware.com/vmware-essential-pks)      |          |[VMware Essential PKS](https://cloud.vmware.com/vmware-essential-pks)
 
 {{% /capture %}}
 
-{{% capture whatsnext %}}
-Go to [Picking the Right Solution](/docs/setup/pick-right-solution/) for a complete
-list of solutions.
-{{% /capture %}}
+
```

Comment:
Do you want a link to `Picking the right solution` or are you planning on removing that page?

Reply:
Thanks!!! Yes, the plan is to remove the link to Picking the right solution page after the content of this page is approved and we decide to delete the Picking the right solution page.  However, I see no harm is removing it even now. removed the link as suggested.

Reply:
removed it as suggested.